mod/Jailbreak.SpecialDay/SpecialDays/OneInTheChamberDay.cs

```diff
@@ -0,0 +1,73 @@
+using CounterStrikeSharp.API;
+using CounterStrikeSharp.API.Core;
+using CounterStrikeSharp.API.Modules.Entities;
+using CounterStrikeSharp.API.Modules.Entities.Constants;
+using Jailbreak.English.SpecialDay;
+using Jailbreak.Formatting.Views;
+using Jailbreak.Public.Extensions;
+using Jailbreak.Public.Mod.SpecialDay;
+using Jailbreak.Public.Mod.SpecialDay.Enums;
+using Jailbreak.Public.Utils;
+
+namespace Jailbreak.SpecialDay.SpecialDays;
+
+public class OneInTheChamberDay(BasePlugin plugin, IServiceProvider provider) 
+  : FFADay(plugin, provider) {
+  
+  public override SDType Type => SDType.OITC;
+  
+  public override ISpecialDayInstanceMessages Messages
+    => new SoloDayMessages("One In The Chamber",
+      "You only have one bullet. Kill someone else to get another bullet. "
```

Comment:
With the new dev push, the description is now a params, so if you want, you could specify multiple strings that would be formatted one per line.